modules/eventing/pages/eventing-language-constructs.adoc

```diff
@@ -392,35 +392,21 @@ Integrating Eventing with Analytics:
 [source,javascript]
 ----
 function OnUpdate(doc, meta) {
-    // Ignore information we don't care about
-    if (doc.type !== 'airline') return;
-
-    // Get the total routes per IATA
-    var route_cnt = 0;     
-    // Uses a true variable as a SQL++ parameter  
-    var airline = doc.iata;  
-
-    var results = ANALYTICS(
-        "SELECT COUNT(*) AS cnt
-        FROM `travel-sample`.`inventory`.`route`
-        WHERE type = \"route\" 
-		AND airline = $1", [doc.iata]
-    );
-
-    // Stream results using the 'for' iterator
-    for (var item of results) {   
-        route_cnt = item.cnt;
-    }
+  var count = 0;
+  const limit = 4;
 
-    // End the query and free the resources held
-    results.close();    
+  let query = couchbase.analyticsQuery('SELECT * FROM default LIMIT $limit;', {"limit": limit});
+  for (let row of query) {
+  ++count;
+  }
 
-    // Log the KEY, AIRLINE and ROUTE_CNT
-    log("key: " + meta.id + ", airline: " + doc.iata + ", route_cnt: " + route_cnt);
+  if (count === limit) {
+  dst_bucket[meta.id] = 'yes';
+  }
 }
```

Comment:
I think we should (re)format the JavaScript code in this example to ensure the indentation is consistent.